### ISSUE TITLE
Apollo 1.4.0 Operation Manifest not generated 

### ISSUE THREAD

**Issue by trevor-holliday-instacart** [apollographql/apollo-ios#3202]
### Summary

I am working on migrating our Apollo version to 1.4.0

I have configured our json config to generate the operation manifest during code generation as the document specifies. Example is given in ruby. 
`
  'operationManifest' => {
        'generateManifestOnCodeGeneration' => true,
        'path' => operation_manifest_path,
        'version' => 'persistedQueries'
      },
      'options' => {
        'operationDocumentFormat' => [
            'operationId'
        ]
      }
` 

The generate command succeeds, but no operation manifest is generated. 

I dont know if this [issue](https://github.com/apollographql/apollo-ios/issues/3199) is related and somehow the `itemsToGenerate` is still only passing in `.code`. 

### Version

1.4.0

### Steps to reproduce the behavior

setup code gen json config

pass in expected operation manifest data 

`
      'operationManifest' => {
        'generateManifestOnCodeGeneration' => true,
        'path' => operation_manifest_path,
        'version' => 'persistedQueries'
      },
      'options' => {
        'operationDocumentFormat' => [
            'operationId'
        ]
      }
`

Observe no operation manifest generated.

**Comment by BobaFetters** [comment]
@trevor-holliday-instacart What path (or path format) are you using for the output path? Wondering if its just not showing up in the location you expect, if the path begins with `./` then it will be relative to you projects root directory, otherwise it is an absolute file system path.

**Comment by AnthonyMDev** [comment]
Hi Trevor, 

I'm not sure why you're building a temp file before generation, when the manifest is generated its going to overwrite any existing file at the same path.

I'm confused about what you are doing here though. It looks like you're trying to use `operation_manifest_path` as a variable of some sort? But this is a json config file, it doesn't evaluate env variables. Can you provide some more information about what you're doing when you "pass the absolute path along"?

**Comment by trevor-holliday-instacart** [comment]
Hello @AnthonyMDev ,

Thank you for the information. I was trying to figure out if the script would write a new file at the specified path or if I needed to pass in a file. Either way when I told the cli to generate the manifest it never happened. 

We have a internal pattern for managing persisted queries. Right now we generate the cli config on the fly, and do not commit it to the repo. I was going to have to do something similar with the operation manifest to honor our internal query persistence. 

We found that we can generate the operation identifiers json similar to how it was done with the legacy system by passing in `operationIdentifiersPath: operations_ids_path` into the config. 

I am not sure if I have an error previously with how I was attempting to setup the operation manifest, but I have found the path forward that I need. We can close this bug if you believe it was user error on my part.
